ros2component/ros2component/verb/load.py

```diff
@@ -35,51 +31,32 @@ class LoadVerb(VerbExtension):
     def add_arguments(self, parser, cli_name):
         add_arguments(parser)
         argument = parser.add_argument(
-            'container_node_name', help='Container node name to load component into'
+            'container_node_name', help='Container node name to unload component from'
         )
         argument.completer = container_node_name_completer
-        argument = parser.add_argument(
-            'package_name', help='Name of the package where the component is to be found'
-        )
-        argument.completer = package_with_components_name_completer
-        argument = parser.add_argument(
-            'plugin_name', help='Type name of the component to be loaded'
-        )
-        argument.completer = ComponentTypeNameCompleter(package_name_key='package_name')
-        parser.add_argument('-n', '--node-name', default=None, help='Component node name')
-        parser.add_argument('--node-namespace', default=None, help='Component node namespace')
-        parser.add_argument('--log-level', default=None, help='Component node log level')
+        add_component_arguments(parser)
         parser.add_argument(
-            '-r', '--remap-rule', action='append', default=None, dest='remap_rules',
-            help="Component node remapping rules, in the 'from:=to' form"
-        )
-        parser.add_argument(
-            '-p', '--parameter', action='append', default=None, dest='parameters',
-            help="Component node parameters, in the 'name:=value' form"
-        )
-        parser.add_argument(
-            '-e', '--extra-argument', action='append', default=None, dest='extra_arguments',
-            help="Extra arguments for the container, in the 'name:=value' form"
+            '-q', '--quiet', action='store_true', default=False,
+            help='Print bare minimum output: component unique IDs and names only'
```

Comment:
Nitpick: I would drop the phrase "bare minimum output" and just state what it does:

> Only print component unique IDs and names

Same below.

Reply:
Sounds good.